benefit/data/sheets/CaseEventToFields/CaseEventToFields.json

```diff
@@ -2195,7 +2195,7 @@
   },
   {
     "CaseEventID": "directionIssued",
-    "CaseFieldID": "reservedToJudge",
+    "CaseFieldID": "reservedToJudgeInterloc",
```

Comment:
#############################
#############################
REVERT THIS BEFORE MERGING OR TESTING WITHOUT TYPEAHEAD FLAG
THIS IS FOR TESTING PURPOSES ONLY
#############################
#############################